index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet"/>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"/> 
+
+    <title>Portfolio</title>
+    
+<body>
+  <header class="container">
+    <a href="#" class="logo">
+        My Logo
+    </a>
+        <nav class="button">
+           <a href="#">
+           <img src="images/icon_image/Union.png" alt="buttom" width="18px" height="12px"> 
+           </a>
+          </nav>
+      </header>
+    <section class="second-container">
+        
+      <div class="header">
+        <h1 class=" bigger-text">I'm Kwamay. Glad to see you!</h1>
+      </div> 
+      <div>
+        <p class="smaller-text">I'am a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      </div> 
+      <h3 class="connect">
+        LET'S CONNECT
+      </h3>
+
+        <ul class="social">
+        <li>
+          <a href="#" class="">
+            <img
+            src="images/icon_image/twitter.png"
+            alt="twitter"
+            height="19.6px"
+            width="19.99px"
+             /></a>
+        </li>
+        <li class="icons-li">
+          <a href="#" class="icons-link">
+            <img
+            src="images/icon_image/linkedin.png"
+            alt="linkedin"
+            height="16.97px"
+            width="12.71"
+            ></a>
+        </li>
+        <li class="icons-li">
+          <a href="#" class="icons-link">
+            <img
+            src="images/icon_image/medium.png"
+            alt="medium"
+            height="16px"
+            width="20px"
+            ></a>
+        </li>
+        <li class="icons-li">
+          <a href="#" class="icons-link">
+            <img
+          src="images/icon_image/github.png"
+          alt="Github"
+          height="20px"
+          width="17.99px"
+          ></a>
+        </li>
+        <li class="icons-li">
+          <a href="#" class="icons-link">
+            <img
+          src="images/icon_image/angelist.png"
+          alt="Hand"
+          height="20px"
+          width="14.7px"
+          ></a>
+        </li>
```

Comment:
- [ ] This is a brilliantly implemented job :+1: but it seems you were executing CSS rules in the HTML file which is not really a best practice. Kindly remove these, and apply the styles using your element tag. class tags, etc. 

- Separation of concerns is very important in code to improve understandability and readability :wink: :muscle: